### ISSUE TITLE
`lang: "de"` breaks ligatures with `font: "New Computer Modern"`

### ISSUE THREAD

**Issue by KaiJan57** [typst/typst#4668]
### Description

```
#set text(
  lang: "de",
  font: "New Computer Modern"
)
Definition
```

No fi-Ligature is shown in pdf :(

### Operating system

Linux

### Typst version

- [X] I am using the latest version of Typst

**Comment by Enivex** [comment]
I seem to recall that this has come up before, and that it's a font issue.

**Comment by EpicEricEE** [comment]
I am in contact with the font maintainer and he is open to applying the patches I sent him.

**Comment by EpicEricEE** [comment]
This is now fixed in the New Computer Modern v6.0 fonts that were [published recently](<https://ctan.org/tex-archive/fonts/newcomputermodern/otf>) (yet to be updated in Typst).
